journal/modernized/32-epistles.adoc

```diff
@@ -658,10 +682,15 @@ and dispensation of the new covenant.
 
 I have travelled under a deep sense and concern in this matter for some time.
 
-William Edmundson.The 12th of the First month, 1695.
+[.signed-section-signature]
+William Edmundson
+
+[.signed-section-context-close]
+The 12th of the First month, 1695.
 
 === To His Son
```

Comment:
Ok, so this one -- let's change it to a `[.letter-heading]`.  I'm still working out all of the kinks and conventions, but for now at least, if we have a heading for a letter that is in the form of "GEORGE FOX TO WILLIAM PENN" or "TO WILLIAM PENN" -- so basically "X to Y" or "to Y", then we'll go with a letter heading.  The thing about letter-heading is that they are not technically _headings_, which means we remove the equals sign.  So it should read like this:

```ASCIIDOC
[.letter-heading]
To His Son
```

If you go here: https://flp-styleguide.netlify.com/#letters and scroll down just a hair, you can see an example.

Don't worry, there's really no way I would have expected you to get this the first time, but I thought it was worth pointing out the reasoning for future intake you're doing. 👍

Reply:
Thanks, Jared! and I do refer to the style guide a lot, I'm sure it doesn't seem like it though!! :)

Reply:
And it is extremely helpful!!